backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc

```diff
@@ -13,3 +13,14 @@ You can create restore hooks to run commands in a container in a pod by editing
 include::modules/oadp-review-backup-restore.adoc[leveloffset=+1]
 include::modules/oadp-creating-restore-cr.adoc[leveloffset=+1]
 include::modules/oadp-creating-restore-hooks.adoc[leveloffset=+1]
+[NOTE]
```

Comment:
I think this "note" should be "important", since it relates to a workaround to restore a backup. Don't want users to miss this information!

Reply:
Done, thanks.